packages/libs/react-ui/src/components/Button/ToggleButton.tsx

```diff
@@ -29,7 +29,7 @@ const ToggleButtonBase = (
 
   const onlyIcon = props.icon !== undefined;
   const content = onlyIcon ? (
-    props.icon
+    <span className={iconOnlyStyle}>{props.icon}</span>
```

Comment:
can this be done within the Button component?

Reply:
Yes I've left it out of scope for now and just fix the styling which was reported by Steven. Wanted to know what the use cases would be for this button.